README.md

```diff
@@ -24,22 +25,23 @@ make GOLDFLAGS="-w -linkmode external -extldflags -static" && docker build .
 
 ## Usage
 
-Command-line options: (Only `-ntp.server` is required.)
+Command-line options: (Only `--ntp.server` is required.)
```

Comment:
Package `flag` uses single dashes for command-line options. Why did you change to double dashes throughout this file?

Reply:
Hi,

Actually [`flags` documentation](https://golang.org/pkg/flag/) states that both single and double dashes are supported.

It's usually a good practice to use single dash for short argument names and double dashes for full argument names (Prometheus [does the same](https://prometheus.io/docs/prometheus/latest/configuration/configuration/)).

So, basically it's just a documentation change, but if you would prefer keeping it as is, I can revert it.

Reply:
> It's usually a good practice to use single dash for short parameter names and double dashes for full parameter names

I'm on board with that as well, but thought the convention is to prefer single dashes when using `flag`. See for example `go help build`, where long options also use single dashes.

I'd prefer if you revert that particular documentation change to avoid the appearance that upgrading users need to change their invocations.

Reply:
Done!